executioncontext/projectview/src/test/resources/projectview/file1.bazelproject

```diff
@@ -23,3 +23,10 @@ build_flags:
     --build_flag1.1=value1.1
     # --legacy_build_flag=old_value
     --build_flag1.2=value1.2
+
+directories:
+  included_dir1
+  included_dir2
+  -excluded_dir1
+
+derive_targets_from_directories: true
```

Comment:
u can update other files as well (take a look how `targets` and some singleton section like `bazel_path` are placed and try to do the same for new sections - tests'll need an update then as well